test

```diff
@@ -1235,6 +1235,59 @@ test_startup_staking_ledgers() {
     teardown
 }
 
+test_watch_staking_ledgers() {
+    test=test_watch_staking_ledgers
+
+    setup
+    idxr_server_start \
+        --block-startup-dir ./blocks \
+        --ledger-watch-dir ./staking-ledgers \
+        --database-dir ./database \
+        --log-dir ./logs \
+        --log-level debug
+    sleep 1
+
+    # copy epoch 0 staking ledger to watched directory
+    cp $STAKING_LEDGERS/mainnet-0-jx7buQVWFLsXTtzRgSxbYcT8EYLS8KCZbLrfDcJxMtyy4thw2Ee.json ./staking-ledgers
+    sleep 1
+
+    pk=B62qiy32p8kAKnny8ZFwoMhYpBppM1DWVCqAPBYNcXnsAHhnfAAuXgg
+    token=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.token)
+    public_key=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.pk)
+    balance=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.balance)
+    delegate=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.delegate)
+    voting_for=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.voting_for)
+    receipt_chain_hash=$(idxr staking-ledger-epoch -e 0 | jq -r .staking_ledger.${pk}.receipt_chain_hash)
```

Comment:
I would only fetch the staking-ledger-epoch once. Write to a file and cat to `jq`